src/devices/Ft232H/Ft2232HDevice.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Iot.Device.Common;
+using Iot.Device.FtCommon;
+
+namespace Iot.Device.Ft2232H
+{
+    /// <summary>
+    /// FT232H Device
+    /// </summary>
+    public class Ft2232HDevice : Ftx232HDevice
+    {
+        /// <summary>
+        /// Gets all the FT232H connected
+        /// </summary>
+        /// <returns>A list of FT232H</returns>
+        public static List<Ft2232HDevice> GetFt2232H()
+        {
+            List<Ft2232HDevice> ft2232s = new List<Ft2232HDevice>();
+            var devices = FtCommon.FtCommon.GetDevices(new FtDeviceType[] { FtDeviceType.Ft2232H });
```

Comment:
The method you're calling here (`FtCommon.GetDevices(FtDeviceType[])`) is kind of weird, because you could  provide the same type multiple times. Maybe use a lambda instead of the list argument? Or check for duplicates?

Reply:
the GetDevices method actually returns only the types you are interested in. Idea like in your previous comment is being able to select just 1 type. Which I'm doing here. But in the previous one, you can select multiple types.
Those are common functions for FT4222 for example as well

Reply:
Sorry if I was unclear. I meant that if you call `FtCommon.FtCommon.GetDevices(new FtDeviceType[] { FtDeviceType.Ft2232H , FtDeviceType.Ft2232H});` you suddenly get two devices. It's silly to do that, but I think it should be prevented.

Reply:
> Sorry if I was unclear. I meant that if you call `FtCommon.FtCommon.GetDevices(new FtDeviceType[] { FtDeviceType.Ft2232H , FtDeviceType.Ft2232H});` you suddenly get two devices. It's silly to do that, but I think it should be prevented.

OK, I see your point. But that's an internal function. Still I can adjust this to make sure the list is deduplicated: https://github.com/dotnet/iot/blob/a0db92f8c766771434095662ce2a0c0e47bec24c/src/devices/FtCommon/FtCommon.cs#L72

Reply:
So I made it unique. Thanks for point out